content/html/concepts/elements/terms/col/col.md

```diff
@@ -0,0 +1,42 @@
+---
+Title: '<col>'
+Description: 'Creates properties for table columns, including styling and attributes, often used within a <colgroup> to apply them to multiple columns simultaneously.'
+Subjects:
+  - 'Web Development'
+  - 'Web Design'
+Tags:
+  - 'Elements'
+  - 'Tags'
+CatalogContent:
+  - 'learn-html'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The **`<col>`** element in HTML is used to define properties for table columns, such as styling or attributes, and is often used within a `<colgroup>` to apply these properties to multiple columns simultaneously.
+
+## Syntax
+
+```pseudo
+<col attributes>
+```
+
+HTML tables are defined by their rows, not by columns. The result of this is that any style applied to a row will override any style applied to column. Only a handful of CSS properties can be controlled via the `<col>` element, such as:
```

Comment:
I understand that in Syntax, you are using `attributes` as a placeholder for different CSS properties listed after. You should mention what `attributes` stands for in the last sentence- here.